src/main/java/is/citizen/sdk/api/CitizenApi.java

```diff
@@ -110,6 +112,84 @@ private void log(int status, String errorMessage) {
         return Optional.empty();
     }
 
+    /**
+     * Convert an authentication public key from the Citizen Service into a Java {@link PublicKey}
+     *
+     * @param publicKeyString authentication public key in Base 64 encoded DER format.
+     *
+     * @return {@link PublicKey} converted public key
+     */
+    public Optional<PublicKey> convertAuthPublicKey(String publicKeyString) {
```

Comment:
Do you think it would be better to make the method name convertStringPublicKeyToJava() ?? It is more descriptive for the user? Same for the private key below. Easier for users to understand what this is doing?

Reply:
Fixed.